Claudia/src/main/java/FirstTask/Month.java

```diff
@@ -0,0 +1,45 @@
+package FirstTask;
+
+public class Month {
+    public static void main(String[] args) {
+        int month = 9;
+
+        switch (month) {
+            case 1:
+                System.out.println("January");
+                break;
+            case 2:
+                System.out.println("February");
+                break;
+            case 3:
+                System.out.println("March");
+                break;
+            case 4:
+                System.out.println("April");
+                break;
+            case 5:
+                System.out.println("May");
+                break;
+            case 6:
+                System.out.println("June");
+                break;
+            case 7:
+                System.out.println("July");
+                break;
+            case 8:
+                System.out.println("August");
+                break;
+            case 9:
+                System.out.println("September");
+                break;
+            case 10:
+                System.out.println("October");
+                break;
+            case 11:
+                System.out.println("November");
+                break;
+            case 12:
+                System.out.println("December");
+        }
+    }
```

Comment:
It's a good approach to extract this logic in a separate method for reusability. For example if I want to print 3rd, 6th and 10th month at the same time.

Reply:
Done